catalog/dags/elasticsearch_cluster/create_proportional_by_source_staging_index/create_proportional_by_source_staging_index_dag.py

```diff
@@ -171,10 +172,14 @@ def create_proportional_by_source_staging_index():
     )
 
     point_alias = es.point_alias(
-        index_name=destination_index_name, alias=destination_alias, es_host=es_host
+        es_host=es_host,
+        target_index=destination_index_name,
+        target_alias=destination_alias,
```

Comment:
Should the param/capacity to remove the old source index be added to this DAG as well?

Reply:
We can! I hadn't because I mistakenly thought the alias generated by this DAG had a timestamp in it and would therefore never be replaced, but that's obviously the index name 😅 Added in.